core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.kt

```diff
@@ -94,6 +94,7 @@ open class ErrorActivity : BaseActivity() {
       putExtra(Intent.EXTRA_TEXT, buildBody())
```

Comment:
we can extract a variable for `buildBody()` so we don't need to construct it again